Ubuntu1804-readme.md

```diff
@@ -1 +1,297 @@
+# Ubuntu 18.04.5 LTS
+- Image Version: 20200820.1
 
+## Installed Software
+### Basic packages
+- dnsutils
+- iproute2
+- iputils-ping
+- libunwind8
+- locales
+- openssh-client
+- tzdata
+- upx
+- zstd
+- libxkbfile-dev
+- pkg-config
+- libsecret-1-dev
+- libxss1
+- libgconf-2-4
```

Comment:
I believe this section should be split to 2 sections: packages and tools.
For example, `yamllint`, `jq`, `zstd` are tools and should be present here.
Packages should be put at the bottom of doc file because they are very useless

Reply:
Done